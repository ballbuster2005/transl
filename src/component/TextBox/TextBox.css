.TextBox {
    margin: 50px;
}