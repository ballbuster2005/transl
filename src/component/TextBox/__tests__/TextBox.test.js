import React, { createElement } from 'react'
import { create } from 'react-test-renderer'

import TextBox from './../TextBox'

describe("[RENDER] TextBox â€“ Snapshot test", ()=> {
    test("Matches the snapshot", () => {
        const textbox = create(<TextBox label="TextBox" id="textbox" value="0"/>)
        expect(textbox.toJSON()).toMatchSnapshot()
    })
})
