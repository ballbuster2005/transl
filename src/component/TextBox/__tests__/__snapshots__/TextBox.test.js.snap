// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[RENDER] TextBox â€“ Snapshot test Matches the snapshot 1`] = `
<div
  class="TextBox"
  className="textbox"
>
  <label
    className="inputLabel"
  >
    TextBox
  </label>
  <input
    className="inputValue"
    id="textbox"
    onChange={[Function]}
    type="text"
    value="0"
  />
</div>
`;
